Bottom: 8217d138ddcb2777e526b8aa184ca7b598b2bf26
Top:    2ab73bb238ed5633cd54a506735f4d0cd946dd7a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-24 14:40:52 +0100

Refresh of demo-of-exporter-specific-expo

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 8270236..4265824 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -696,5 +696,5 @@ common_export_region (HID *hid, BoxType *region)
     //PrintAssembly (region, solder_group, 1);
 
   if (gui->set_layer ("fab", SL (FAB, 0), 0))
-    PrintFab ();
+    PrintFab (Output.fgGC);
 }
diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 7710350..bca08e3 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -585,7 +585,7 @@ gerber_expose (HID * hid, BoxType *drawn_area, void *item)
     DrawPaste (SOLDER_LAYER, drawn_area);
 
   if (gui->set_layer ("fab", SL (FAB, 0), 0))
-    PrintFab ();
+    PrintFab (Output.fgGC);
 
   gui->destroy_gc (Output.fgGC);
   gui->destroy_gc (Output.bgGC);
